app/views/tea_times/index.html.erb

```diff
@@ -0,0 +1,50 @@
+<% next_month_name = Date.today.next_month.strftime("%B") %>
+
+<div class="cover-photo" style="background-image: linear-gradient(to bottom, rgba(247, 191, 77, 0.87), rgba(247, 191, 77, 0.87)), url('<%= image_path("missing-city-image.jpg") %>'); margin-bottom: 50px;">
```

Comment:
Styles are inline because @ankitshah811 will do the CSS and I don't want to create classes that might be rewritten later.